ibis/pyspark/api.py

```diff
@@ -1,4 +1,5 @@
 from ibis.pyspark.client import PySparkClient
+from ibis.pyspark.compiler import dialect  # noqa: F401
```

Comment:
why does this get put in the public api?

Reply:
It is because this line here:
https://github.com/ibis-project/ibis/blob/master/ibis/tests/backends.py#L145